### PR TITLE
Py2 Deprecation

### PR DESCRIPTION
**Pull request by markgalpin**
Are we ready for this?

### CODE REVIEW

File: introduction.rst [hunk 1]
```diff
@@ -118,7 +118,20 @@ This means that public repositories, like conan-center assume the use of the lat
 
 If you have any question regarding Conan updates, stability, or any clarification about this definition of stability, please report in the documentation issue tracker: https://github.com/conan-io/docs.
 
-
+.. _python2:
+
+Python 2 Deprecation Notice
+---------------------------
+Before 1.6, all Conan features are fully supported in both Python2 and Python3.
```
Comment (danimtb) [inline introduction.rst]: **were** fully

**Comment by memsharded** [comment]
I agree, I think this should go in 2 places:
- The "install" section in the docs
- A link in the downloads page in conan.io

**Comment by markgalpin** [comment]
I put it here (there's already a warning in Install, which I may strengthen later) because it feels like it impacts/modifies the stability commitment.  If we don't think so, I can move this other places.

**Comment by memsharded** [comment]
I think the main point should be the installation section, and in any case, link it briefly from the stability subsection of the introduction.

**Pull request merged**